### PR TITLE
JWKS Response as JSON

### PR DESCRIPTION
**Pull request by shoebkhan09**

### CODE REVIEW

File: oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java [hunk 1]
```diff
@@ -196,7 +226,8 @@ public String getRp(@HeaderParam("Authorization") String authorization, String p
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
     public String getJwks(@HeaderParam("Authorization") String authorization, String params) {
-        return process(CommandType.GET_JWKS, params, GetJwksParams.class, authorization);
+        final ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL,true);
+        return processWithCustomMapper(CommandType.GET_JWKS, params, GetJwksParams.class, authorization, mapper);
```
Comment (yuriyz) [inline oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java]: Now we have 2 methods for processing commands, lets keep single method and pass mapper inside the `process`, 
```
private static <T extends IParams> String process(ObjectMapper mapper, CommandType commandType, String paramsAsString, Class<T> paramsClass, String authorization) {
    ...
}
```

Also overload `process` method with mapper that suites for 90% of commands, like 
```
private static <T extends IParams> String process(ObjectMapper mapper, CommandType commandType, String paramsAsString, Class<T> paramsClass, String authorization) {
    return process(CoreUtils.createJsonMapper(), commandType, paramsAsString, paramsClass, authorization);
}
```

File: oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java [hunk 2]
```diff
@@ -210,6 +241,29 @@ public String getJwks(@HeaderParam("Authorization") String authorization, String
...
+        Object forJsonConversion = getObjectForJsonConversion(commandType, paramsAsString, paramsClass, authorization);
+        final String json = CoreUtils.asJsonSilently(forJsonConversion);
+        LOG.trace("Send back response: {}", json);
+        return json;
+    }
+
+    /*
+    Writes JSON using specified object mapper.
+     */
+    private static <T extends IParams> String processWithCustomMapper(CommandType commandType, String paramsAsString, Class<T> paramsClass, String authorization, ObjectMapper objectMapper) {
```
Comment (yuriyz) [inline oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java]: please remove this method and overload instead as explained above.

File: oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java (hunk 1 above)
Comment (shoebkhan09) [reply oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java]: The two mappers are from different package. If we migrate CoreUtils mapper to new one,  it break some other code.

Comment (yuriyz) [reply oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java]: we must stick to one single mapper from oxauth-client. We don't want to clash with it.

**Pull request merged**